### PR TITLE
store: use raft base tick for max slow log

### PR DESCRIPTION
**Pull request by siddontang**
Origin 1s is too long for slow log.
In raftstore, we will handle raft tick in every `raft_base_tick_interval`, if the operation takes too long > `raft_base_tick_interval`,  the raft is not accurate. 

@BusyJay @hhkbp2

### CODE REVIEW

File: src/raftstore/store/store.rs [hunk 1]
```diff
@@ -554,7 +554,7 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         self.insert_peer_cache(msg.take_to_peer());
 
         let peer = self.region_peers.get_mut(&region_id).unwrap();
-        let timer = SlowTimer::new();
+        let timer = SlowTimer::from_millis(self.cfg.raft_base_tick_interval);
```
Comment (ngaut) [inline src/raftstore/store/store.rs]: I prefer to define a new constant. It's wired to use raft_base_tick_interval

**Comment by ngaut** [comment]
LGTM

**Comment by hhkbp2** [comment]
IMO, the measure of the time taken for a slow operation should not be bound to raft tick. It should be a independent config item.
The raft tick is notably inaccurate even it takes raft_base_tick_interval/2 or raft_base_tick_interval/3 on every tick.

**Comment by siddontang** [comment]
> It should be an independent config item

If we use another field, we will still assign the raft tick value to this field now, so I use a function.

The purpose of the PR is to know exactly why leader lost when we meet 4 or 5 times 1s slow log. We can't explain the phenomenon before, but yesterday I figured it out, so I will close this PR now.

**Pull request closed without merging**